frontend/assets/javascripts/src/modules/form/billingPeriodChoice.js

```diff
@@ -2,10 +2,10 @@ define(
     [
         '$',
         'bean',
-        'lodash/string/template',
+        'lodash/template',
         'src/modules/form/submitButton',
         'src/modules/form/ongoingCardPayments',
-        'text!src/templates/checkout/billingPeriodChoice.html'
+        'text-loader!src/templates/checkout/billingPeriodChoice.html'
```

Comment:
In webpack 2 the automatic `-loader` module name extension is being removed
(https://webpack.js.org/guides/migrating/#automatic-loader-module-name-extension-removed)